### PR TITLE
Ph/add integration tests

### PR DESCRIPTION
**Pull request by p-offtermatt**
- [x] Tests added for any new code
- [x] Ran `make fmt-fix` (or had formatting run automatically on all files edited)

Draft PR for integration tests for https://github.com/informalsystems/apalache/pull/1810

Will be set to ready-for-review once the other PR is merged

[unreleased]: https://github.com/informalsystems/apalache/tree/unstable/.unreleased

### CODE REVIEW

**Comment by rodrigo7491** [comment]
@p-offtermatt Are you sure you linked the right issue?

**Comment by p-offtermatt** [comment]
> @p-offtermatt Are you sure you linked the right issue?

Sorry! I mistyped a 1 as a 2, it should have referred to https://github.com/informalsystems/apalache/pull/1810!
Thanks for pointing out

File: test/tla/BoxDiamond.tla [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---- MODULE BoxDiamond ----
+
+\* Tests behaviour when a diamond operator is followed by a box operator
```
Comment (shonfeder) [inline test/tla/BoxDiamond.tla:3]:
```suggestion
\* Tests behaviour when a box operator is followed by a diamond operator
```

**Review by shonfeder — APPROVED** [review]
LGTM! Just found one typo.

**Review by konnov — APPROVED** [review]
Looks good.

**Pull request merged**